packages/zowe-explorer-api/__tests__/__unit__/profiles/ProfilesCache.unit.test.ts

```diff
@@ -87,12 +87,11 @@ function createProfInfoMock(profiles: Partial<imperative.IProfileLoaded>[]): imp
     const teamConfigApi: Partial<imperative.Config> = {
         api: {
             profiles: {
-                get: jest.fn(),
+                get: jest.fn().mockReturnValue({}),
```

Comment:
I think we need it to be a `jest.fn` because we do this in some tests:
`mocked(profInfoMock.getTeamConfig().api.profiles.get).mockReturnValueOnce({ tokenType: imperative.SessConstants.TOKEN_TYPE_JWT });`